enhancements/sig-architecture/30-clusterset-override/README.md

```diff
@@ -0,0 +1,279 @@
+
+In this proposal, managedClusterSet could use any labels to select managedClusters. And if I add a label for a managedCluster, it may lead to the managedCluster added to a managedClusterSet. So we need to rethink about the rbac control of managedClusterSet
+
+We will use the following questions to consider the RBAC control.
+
+#### What purposes will managedClusterSet may be used for
+a. Create a managedClusterSet for disaster recovery purposes, which should include specific clusters.
+b. Create a managedClusterSet which should include all clusters in one region or cloud provider.
+c. Create a managedClusterSet which has the same capacity(like: all clusters enabled middleware).
+d. Create a managedClusterSet for each squad for resource group purposes.
+
+#### Who can create a managedClusterSet
+Same as the current way, Anyone who has `create` permission to `managedclustersets` could create managedClusterSet. 
+But generally, Cluster admin should not give this permission to others. So only cluster admin could create the managedClusterSet.
+
+#### What kinds of roles exist related to managedClusterSets and managedClusters
+In General, there are three kinds of roles related to managedClusterSet.
+1. Cluster admin
+   - Cluster admin could `create`, `get`, `list`, `update`, `delete` all managedClusterSets.
+   - Cluster admin could `bind` all managedClusterSets to all namespaces.
+   - Cluster admin could `create`, `get`, `list`, `update`, `delete` all managedClusters.
+   - Only Cluster admin could `accept` managedClusters.
+   
+2. Controller/agent which run in managedClusters
+   - The Controller/agent should not `create`, `get`, `list`, `update`, `delete` managedClusterSets.
+   - The Controller/agent should not `bind` managedClusterSet.
+   - The Controller/agent only should `get` related managedCluster and `update` the managedCluster status and labels.
+
+3. Non Cluster admin users
+   - These users should `get` some managedClusterSets, and could not `create`, `update`, `delete`, `list` managedClusterSets.
+   - These users should `bind` some managedClusterSets to their namespaces.
+   - These users may need to `create` managedClusters(For Resource Group Purpose managedClusterSet[d], in each group, the group members may need to create the managedClusters). They could also `delete` these managedClusters, and `update` these managedClusters labels.
+
+#### Who can join a managedCluster to a managedClusterSet
+In this proposal, managedClusterSet could use any labels to select managedClusters. If one user adds a label to a managedClusters, it may lead to this managedClusters added to a managedClusterSet. So this question also means "who can add/update a label for managedClusters".
+
+1. Cluster admin
+Cluster admin could add any labels to any managedClusters.
+
+2. Controller/agent which run in managedClusters
+The controller/agent may add labels like: `region: apac`, `cloud: Amazon`, `middlewareEnabled: true`
```

Comment:
Is this the case currently or the case after this proposal?  Is there a specific prefix reserved?  should there be?

Reply:
This is the case currently.
After the proposal, if user want to restrict the label, he/she could add labels like: `cluster.open-cluster-management.io/region:apac`. And add the permission to controller/agent
`
 - apiGroups: ["cluster.open-cluster-management.io"]
   resources: ["managedclusters/label"]
   resourceNames: ["cluster.open-cluster-management.io/region:*"]
   verbs: ["create"]
`